lib/plugins/backup.rb

```diff
@@ -3,8 +3,8 @@
 require_relative '../moonshot/creds_helper'
 
 module Moonshot::Plugins # rubocop:disable Documentation
-  # Monkeypatching TarWriter for our
-  # needs to improve clarity
+  # Defining new method for the ease of use in this
```

Comment:
I'm sure I had a great reason, the code has evolved so much ever since that I don't know what it was. Patch removed.